pkg/ipamd/datastore/data_store_test.go

```diff
@@ -350,7 +350,7 @@ func TestPodIPv4Address(t *testing.T) {
 	assert.Equal(t, ds.assigned, 2)
 }
 
-func TestGetStats(t *testing.T) {
+func TestGetIPStatsV4(t *testing.T) {
```

Comment:
v4 and v6 can be combined into 1 test

Reply:
Each of test cases inits its own DataStore with custom configuration, feels like having separate tests is a bit cleaner.
Would you like to merge them back together?

Reply:
Okay I see there are few more configurations